src/framework/components/collision/trigger.js

```diff
@@ -88,11 +89,12 @@ Object.assign(pc, function () {
             var body = this.body;
             if (body) {
                 var position = this.entity.getPosition();
+                var offsetPosition = position + data.offset;
```

Comment:
Same again. `pc.Vec3` math does not work using operator overloading - use the API functions instead.